src/vaults/apys/implementations/idle-finance.js

```diff
@@ -12,6 +12,8 @@ const uniswapMethods = require('../../../lib/web3/contracts/uniswap/methods')
 const { idleLendingToken, idleController } = require('../../../lib/web3/contracts')
 const { getTokenPrice } = require('../../../prices')
 
+const addresses = require('../../../../data/mainnet/addresses.json')
```

Comment:
Use `src/lib/data/addresses.json` file here is okay. Add the address to both `address.json` files.